test/ClangImporter/attr-swift_name.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %empty-directory(%t.mcp)
-// RUN: not %target-swift-frontend(mock-sdk: %clang-importer-sdk) -I %S/Inputs/custom-modules -Xcc -w -typecheck %s -module-cache-path %t.mcp -disable-named-lazy-member-loading 2>&1 | %FileCheck %s
+// RUN: not %target-swift-frontend(mock-sdk: %clang-importer-sdk) -disable-experimental-clang-importer-diagnostics -I %S/Inputs/custom-modules -Xcc -w -typecheck %s -module-cache-path %t.mcp -disable-named-lazy-member-loading 2>&1 | %FileCheck %s
```

Comment:
Why did this test need to be updated?

Reply:
It doesn't need updating anymore, but at the time it was because the new diagnostics were tripping a CHECK-NOT. The diagnostic was accurate, and is still there, but now it's emmitted later in a position that does not trip a CHECK-NOT.

The new diagnostic informs about a circularFriends method that can't be imported because parameter type `CircularName` is unimportable. Not exactly sure of the details, but looks intentional, I can dig deeper if need be.